### ISSUE TITLE
[Question] Can i adjust the learning rate of CRF?

### ISSUE THREAD

**Issue by cccccrj** [BrikerMan/Kashgari#482]
**You must follow the issue template and provide as much information as possible. otherwise, this issue will be closed.
请按照 issue 模板要求填写信息。如果没有按照 issue 模板填写，将会忽略并关闭这个 issue**

## Check List
Thanks for considering to open an issue. Before you submit your issue, please confirm these boxes are checked.

**You can post pictures, but if specific text or code is required to reproduce the issue, please provide the text in a plain text format for easy copy/paste.**

- [x] I have searched in [existing issues](https://github.com/BrikerMan/Kashgari/issues?utf8=%E2%9C%93&q=is%3Aissue+) but did not find the same one.
- [x] I have read the [documents](https://kashgari.bmio.net)

## Environment

- OS [e.g. Mac OS, Linux]:
- Python Version: py3.6
- requirements.txt:
- tf2.2
- keras2.3
- kashgari-2.0.2

```

## Question
请问有参数可以直接调整CRF层的学习率么？
